src/main/java/database/sql/ddl/EntityClassInspector.java

```diff
@@ -0,0 +1,82 @@
+package database.sql.ddl;
+
+import jakarta.persistence.*;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class EntityClassInspector {
+    private final Class<?> entityClass;
+
+    public EntityClassInspector(Class<?> entityClass) {
+        this.entityClass = entityClass;
+    }
+
+    public String getTableName() {
+        Table tableAnnotation = entityClass.getAnnotation(Table.class);
+        if (tableAnnotation != null && !tableAnnotation.name().isEmpty()) {
+            return tableAnnotation.name();
+        } else {
+            return entityClass.getSimpleName();
+        }
+    }
+
+    public Stream<EntityColumn> getVisibleColumns() {
+        return Arrays.stream(this.entityClass.getDeclaredFields())
+                .filter(this::notTransientField)
+                .map(this::getEntityColumn);
+    }
+
+    private boolean notTransientField(Field field) {
+        return field.getAnnotation(Transient.class) == null;
+    }
+
+    private EntityColumn getEntityColumn(Field field) {
+        Column columnAnnotation = field.getAnnotation(Column.class);
+        Id idAnnotation = field.getAnnotation(Id.class);
+        GeneratedValue GeneratedValueAnnotation = field.getAnnotation(GeneratedValue.class);
+        EntityFieldTypeConverter entityFieldTypeConverter = new MysqlEntityFieldTypeConverter();
+
+        return new EntityColumn(
+                columnName(columnAnnotation, field.getName()),
+                field.getType(),
+                columnLength(columnAnnotation, 255),
+                idAnnotation != null,
+                isAutoIncrement(GeneratedValueAnnotation, false),
+                isNullable(columnAnnotation, true),
+                entityFieldTypeConverter);
+    }
+
+    private String columnName(Column columnAnnotation, String defaultName) {
+        if (columnAnnotation != null && !columnAnnotation.name().isEmpty()) {
+            return columnAnnotation.name();
+        } else {
+            return defaultName;
+        }
+    }
+
+    private Integer columnLength(Column columnAnnotation, Integer defaultValue) {
+        if (columnAnnotation != null) {
+            return columnAnnotation.length();
+        } else {
+            return defaultValue;
+        }
+    }
```

Comment:
```suggestion
    private Integer columnLength(Column columnAnnotation, Integer defaultValue) {
        if (columnAnnotation != null) {
            return columnAnnotation.length();
        }
        return DEFAULT_COLUMN_LENGTH;
    }
```
 로 null이 아니면 항상 default값을 반환하고 있어 따로 파라미터로 받지 않아도 될 것 같아요. 혹은 `Column`에서 length default값을 가져오는 방법도 있을 것 같구요.